### DISCUSSION TITLE
Possible to prevent output of default sitemap index file?

### DISCUSSION THREAD

**Discussion by prblackburn** [iamvishnusankar/next-sitemap#334] [Q&A] [answered]
I'm using `getServerSideSitemapIndex` to generate a `server-sitemap-index.xml` that includes both the statically generated sitemap (defaulting as `sitemap-0.xml`) as well as the `server-sitemap.xml` for my dynamic, CMS-based routes.

However, because there is a default sitemap index called `sitemap.xml` being output as well, this means I end up with two visitable sitemap index ~files~ paths: `example.com/sitemap.xml` and `example.com/server-sitemap-index.xml`. 

I also end up with `Sitemap: https://example.com/sitemap.xml` in my `robots.txt` whether I want it or not.

So my question is if it is possible to prevent the default output of `sitemap.xml` sitemap index file and prevent writing `Sitemap: https://example.com/sitemap.xml` to the `robots.txt`?

Apologies if this is somewhere in the docs - it's been a long week and I haven't spotted anything yet!

Update:
I actually think if I can just add the SSR sitemap url `example.com/server-sitemap-index.xml` to the original `sitemap.xml` so that it doesn't only include `sitemap-0.xml`, this would probably solve my problem. Is this doable?

Many thanks,
Paul

**Comment by iamvishnusankar** [comment, marked as answer]
@prblackburn This is resolved in `next-sitemap@v3`

Read: https://github.com/iamvishnusankar/next-sitemap#index-sitemaps-optional

**Reply by ezbeazy** [reply]
using next-sitemap 4.2.2, generateIndexSitemap: false in next-sitemap.config but getting 2 duplicate xml files. 

Am I misunderstanding the generateIndexSitemap option? I assumed this would result in a single xml file, instead of a index sitemap with additional, segmented by size, sitemap files.

**Reply by xning-lci** [reply]
Hi Bro, have you found out the issue?

@iamvishnusankar  I don't think you resolved this issue. Not answering the question

**Reply by xning-lci** [reply]
I am using transform to prevent
 transform: async (config, path) => { return null },
